accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceservice/impl/AceServiceImpl.java

```diff
@@ -591,7 +592,12 @@ private String purgeAuthorizables(Set<String> authorizableIds,
         try {
             JackrabbitSession js = (JackrabbitSession) session;
             UserManager userManager = js.getUserManager();
-            userManager.autoSave(false);
+            try {
+              userManager.autoSave(false);
+            } catch (final UnsupportedRepositoryOperationException e) {
```

Comment:
do you mean just leave only this?

            if (userManager.isAutoSave()) {
                userManager.autoSave(false);
            }

or 

       if (userManager.isAutoSave()) {
            try {
              userManager.autoSave(false);
            } catch (final UnsupportedRepositoryOperationException e) {
              // check added for AEM 6.0
              LOG.warn("disabling autoSave not possible with this user manager!");
            }
        }